bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyComponents.java

```diff
@@ -543,7 +577,9 @@ public static boolean updateSensors(ShellyThingInterface thingHandler, ShellySet
         }
 
         // Update Add-On channeös
-        if (status.extTemperature != null) {
+        if (status.extTemperature != null)
+
+        {
```

Comment:
Something weird happened here.
```suggestion
        if (status.extTemperature != null) {
```